SPECS/libcontainers-common/libcontainers-common.spec

```diff
@@ -70,7 +71,9 @@ github.com/containers libraries, such as Buildah, CRI-O, Podman and Skopeo.
 
 %setup -q -T -D -b 9 -n common-%{commonver}
 %patch 0 -p1
+%patch 1 -p1
```

Comment:
This patch is being re-applied.  First application is on line 70.

Reply:
there was a merge conflict between two patches. both of them were patch 1. Made one of them 1 another 2. I will remove this line.

Reply:
This is from my side.